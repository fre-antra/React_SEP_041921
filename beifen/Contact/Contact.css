.projects {
  background-color: #59585d;
  padding: 50px;
  color: white;
}

.projects-header-quote {
  margin-bottom: 20px;
}
.projects h1,
.projects h3 {
  font-family: "Source Sans Pro", Arial, sans-serif;
}
.projects p {
  font-family: "Roboto", Helvetica, serif;
  padding: 0 10px;
}

.projects-header-quote {
  font-style: italic;
}
/*.luvtalk-project, .personal-website-project, .strike-zone-project, .resume {border: 3px solid white;}*/
.luvtalk-project {
  /* background: linear-gradient(rgba(0, 0, 0, 0.85), rgba(0, 0, 0, 0.85)),
      url(./images/luvtalklogo.png); */
  background-position: center center;
}

.personal-website-project {
  /* background: linear-gradient(rgba(0, 0, 0, 0.85), rgba(0, 0, 0, 0.85)),
      url(./images/logo.svg); */
  background-position: center center;
}

.strike-zone-project {
  /* background: linear-gradient(rgba(0, 0, 0, 0.85), rgba(0, 0, 0, 0.85)),
      url(./images/mlbproject.png); */
  background-position: center center;
}

.resume {
  padding: 50px 0px;
  font-family: "Calisto MT", Helvetica, serif;
}

.resume-header {
  margin-bottom: 20px;
}
.objective h4,
.education h4,
.work-experience h4,
.resume-projects h4,
.activities h4,
.skills h4,
.endorsements h4 {
  border-bottom: 1px solid black;
}
.contact {
  background-color: #3a3a3a;
  padding: 50px 150px;
  color: white;
}
.contact input,
textarea {
  color: black;
}

.contact h1,
.contact h2,
.contact h4 {
  font-family: "Source Sans Pro", Arial, sans-serif;
}
.contact p {
  font-family: "Roboto", Helvetica, serif;
}
.contact-header-quote {
  font-style: italic;
}
.contact-header-quote h4 {
  margin: 5px 0;
}
.contact-instructions,
.contact-form-body {
  margin-top: 50px;
  padding: 0px 6%;
}

.email,
.message {
  margin: 15px 0px;
}
.contact input,
.contact textarea {
  width: 100%;
  padding: 10px;
  margin: 3px 0px;
}

.submit-contact .btn-border {
  margin-top: 10px;
}
