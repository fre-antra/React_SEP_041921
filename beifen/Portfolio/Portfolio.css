.projects {
  background-color: #59585d;
  padding: 50px;
  color: white;
}

.projects-header-quote {
  margin-bottom: 20px;
}
.projects h1,
.projects h3 {
  font-family: "Source Sans Pro", Arial, sans-serif;
}
.projects p {
  font-family: "Roboto", Helvetica, serif;
  padding: 0 10px;
}

.projects-header-quote {
  font-style: italic;
}
/*.luvtalk-project, .personal-website-project, .strike-zone-project, .resume {border: 3px solid white;}*/
.luvtalk-project {
  background: linear-gradient(rgba(0, 0, 0, 0.85), rgba(0, 0, 0, 0.85)),
    url(../Images/luvtalklogo.png);
  background-position: center center;
}

.personal-website-project {
  background: linear-gradient(rgba(0, 0, 0, 0.85), rgba(0, 0, 0, 0.85)),
    url(../Images/logo.svg);
  background-position: center center;
}

.strike-zone-project {
  background: linear-gradient(rgba(0, 0, 0, 0.85), rgba(0, 0, 0, 0.85)),
    url(../Images/mlbproject.png);
  background-position: center center;
}

.resume {
  padding: 50px 0px;
  font-family: "Calisto MT", Helvetica, serif;
}

.resume-header {
  margin-bottom: 20px;
}
.objective h4,
.education h4,
.work-experience h4,
.resume-projects h4,
.activities h4,
.skills h4,
.endorsements h4 {
  border-bottom: 1px solid black;
}
