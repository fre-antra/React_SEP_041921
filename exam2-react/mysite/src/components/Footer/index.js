import { Link } from "react-router-dom";
import fb from "../../images/facebook.svg";
import linkedin from "../../images/linkedin.svg";
import pinterest from "../../images/pinterest.svg";
import email from "../../images/email.svg";
// import fbimg2 from  ""
const Footer = () => {
  return (
    <div className="footer">
      <div className="container-fluid">
        <div className="row">
          <div className="media-links">
            <div className="col-sm-1 col-md-1 col-sm-offset-4 col-md-offset-4">
              <div className="facebook">
                <Link to="#">
                  <img className="img-responsive" src={fb} />
                </Link>
              </div>
            </div>

            <div className="col-sm-1 col-md-1">
              <div className="linkedin">
                <Link to="#">
                  <img className="img-responsive" src={linkedin} />
                </Link>
              </div>
            </div>

            <div className="col-sm-1 col-md-1">
              <div className="pinterest">
                <Link to="#">
                  <img className="img-responsive" src={pinterest} />
                </Link>
              </div>
            </div>

            <div className="col-sm-1 col-md-1">
              <div className="email-icon">
                <Link to="#">
                  <img className="img-responsive" src={email} />
                </Link>
              </div>
            </div>
          </div>
        </div>

        <div className="row">
          <div className="col-sm-6 col-md-6 text-right">
            <div className="personal-contact phone">
              <h4>Phone: XXXXXX</h4>
            </div>
          </div>

          <div className="col-sm-6 col-md-6 text-left">
            <div className="personal-contact email-personal-contact">
              <h4>Email: XXXXX@gmail.com</h4>
            </div>
          </div>
        </div>

        <div className="row">
          <div className="col-sm-4 col-md-4 text-center">
            <div className="company-information">
              <h3>XXXX</h3>
              <h4>Copyright &copy; 2021</h4>
              <h4>NY</h4>
            </div>
          </div>

          <div className="col-sm-2 col-md-2 text-center">
            <div className="home-footer">
              <h4>
                <Link to="#">Home</Link>
              </h4>
            </div>
          </div>

          <div className="col-sm-2 col-md-2 text-center">
            <div className="portfolio-footer">
              <h4>
                <Link to="#">Portfolio</Link>
              </h4>
              <ul>
                <li>
                  <Link to="#">Project1</Link>
                </li>
                <li>
                  <Link to="#">Project2</Link>
                </li>
                <li>
                  <Link to="#">Project3</Link>
                </li>
                <li>
                  <Link to="#">See All</Link>
                </li>
              </ul>
            </div>
          </div>

          <div className="col-sm-2 col-md-2 text-center">
            <div className="resume-footer">
              <h4>
                <Link to="#">Resume</Link>
              </h4>
              <ul>
                <li>
                  <Link to="#">Download</Link>
                </li>
              </ul>
            </div>
          </div>

          <div className="col-sm-2 col-md-2 text-center">
            <div className="contact-footer">
              <h4>
                <Link to="#">Contact</Link>
              </h4>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default Footer;
