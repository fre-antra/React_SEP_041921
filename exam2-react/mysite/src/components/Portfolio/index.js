import { useEffect } from "react";
import $ from "jquery";
const Portfolio = () => {
  useEffect(() => {
    projects();
  }, []);

  function projects() {
    $(".luvtalk-project").css("height", $(".luvtalk-project").width());
    $(".luvtalk-project h3").css(
      "padding-top",
      ($(".luvtalk-project").height() -
        $(".luvtalk-project h3").height() -
        $(".luvtalk-project .row").outerHeight()) /
        2
    );
    $(".personal-website-project").css(
      "height",
      $(".personal-website-project").width()
    );
    $(".personal-website-project h3").css(
      "padding-top",
      ($(".personal-website-project").height() -
        $(".personal-website-project h3").height() -
        $(".personal-website-project .row").outerHeight()) /
        2
    );
    $(".strike-zone-project").css("height", $(".strike-zone-project").width());
    $(".strike-zone-project h3").css(
      "padding-top",
      ($(".strike-zone-project").height() -
        $(".strike-zone-project h3").height() -
        $(".strike-zone-project .row").outerHeight()) /
        2
    );

    var doit;
    $(window).resize(function () {
      clearTimeout(doit);
      doit = setTimeout(projects, 500);
    });
  }

  return (
    <div className="projects" id="#projects">
      <div className="container-fluid">
        <div className="row">
          <div className="col-sm-12 col-md-12 text-center">
            <div className="projects-header">
              <h1>Projects and Portfolio</h1>
            </div>
          </div>
        </div>

        <div className="row">
          <div className="col-sm-12 col-md-12 text-center">
            <div className="projects-header-quote">
              <h3>Sharing my endeavors and passions...</h3>
            </div>
          </div>
        </div>

        <div className="row">
          <div className="col-sm-4 col-md-4 text-center">
            <div className="luvtalk-project">
              <h3>LUV TALK Website</h3>
              <p>
                Website, created during my first internship, which utilized
                HTML, CSS, PHP, Ajax, Javascript/jQuery, and Wordpress.{" "}
              </p>
              <div className="row">
                <div className="col-sm-4 col-md-4 col-sm-offset-4 col-md-offset-4">
                  <div className="project-more-info">
                    <a className="btn btn-default btn-border" href="#">
                      More
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>

          <div className="col-sm-4 col-md-4 text-center">
            <div className="personal-website-project">
              <h3>Personal Website</h3>
              <p>
                Enjoyable side project that was created to experiment with more
                HTML and CSS, but also provided an outlet to showcase my
                abilities and interests.
              </p>
              <div className="row">
                <div className="col-sm-4 col-md-4 col-sm-offset-4 col-md-offset-4">
                  <div className="project-more-info">
                    <a className="btn btn-default btn-border" href="#">
                      More
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>

          <div className="col-sm-4 col-md-4 text-center">
            <div className="strike-zone-project">
              <h3>Strike Zone Analysis</h3>
              <p>
                Data analytics project completed during my time at the Illinois
                Math and Science Academy which studied the baseball strike zone
                based on the state of the game.
              </p>
              <div className="row">
                <div className="col-sm-4 col-md-4 col-sm-offset-4 col-md-offset-4">
                  <div className="project-more-info">
                    <a className="btn btn-default btn-border" href="#">
                      More
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>

        <div className="row">
          <div className="col-sm-2 col-md-2 col-sm-offset-5 col-md-offset-5">
            <div className="see-more-button">
              <a className="btn btn-default btn-border" href="#">
                More Projects
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default Portfolio;
