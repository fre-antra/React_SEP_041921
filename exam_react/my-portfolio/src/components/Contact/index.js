import React, { Component, Fragment } from 'react';

export default class Contact extends Component {
    render() {
        return (
            <div>
                <div className="contact" id="contact">
                    <div className="container-fluid">
                        <div className="row">
                            <div className="col-sm-12 col-md-12 text-center">
                                <div className="contact-header">
                                    <h1>Contact</h1>
                                </div>
                            </div>
                        </div>

                        <div className="row">
                            <div className="col-sm-12 col-md-12 text-center">
                                <div className="contact-header-quote">
                                    <h4>"XXXXX"</h4>
                                    <h4>-XXX</h4>
                                </div>
                            </div>
                        </div>

                        <div className="row">
                            <div className="col-sm-6 col-md-6">
                                <div className="contact-instructions">
                                    <h2>Reaching out to me</h2>
                                    <p>Don't be afraid to contact me! I'm happy to answer any questions, provide more information, or just have a nice conversation! Fill out the form below to being reaching out to me. If you prefer another email client other than your default, which will appear after clicking the button and have all of the information you just entered, you can email me at XXXXX.</p>
                                    <h2>Contact Information</h2>
                                    <p>XXXX</p>
                                    <p>XXXX</p>
                                    <p>(XXXX)-445-7747</p>
                                    <p>XXXXXX@gmail.com</p>
                                </div>
                            </div>

                            <div className="col-sm-6 col-md-6">
                                <div className="contact-form-body">
                                    <form>
                                        <div className="row">
                                            <div className="col-sm-6 col-md-6">
                                                <div className="firstname">
                                                    <h4>Full Name *</h4>
                                                    <input type="text" placeholder="First Name" />
                                                </div>
                                            </div>

                                            <div className="col-sm-6 col-md-6">
                                                <div className="lastname">
                                                    <h4>Full Name *</h4>
                                                    <input type="text" placeholder="Last Name" />
                                                </div>
                                            </div>
                                        </div>

                                        <div className="row">
                                            <div className="col-sm-12 col-md-12">
                                                <div className="email">
                                                    <h4>Email *</h4>
                                                    <input type="text" placeholder="Email" />
                                                </div>
                                            </div>
                                        </div>

                                        <div className="row">
                                            <div className="col-sm-12 col-md-12">
                                                <div className="message">
                                                    <h4>Message *</h4>
                                                    <textarea placeholder="Message"></textarea>
                                                </div>
                                            </div>
                                        </div>

                                        <div className="row">
                                            <div className="col-sm-4 col-md-4 col-sm-offset-4 col-md-offset-4">
                                                <div className="submit-contact">
                                                    <a className="btn btn-default btn-border" href="#">Submit</a>
                                                </div>
                                            </div>
                                        </div>
                                    </form>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        )
    }
}
