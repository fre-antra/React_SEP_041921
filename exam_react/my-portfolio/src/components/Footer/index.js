import React, { Component } from 'react';
import './index.css';

export default class Footer extends Component {
    render() {
        return (
            <div className="footer">
                footer content
            </div>
        )
    }
}
