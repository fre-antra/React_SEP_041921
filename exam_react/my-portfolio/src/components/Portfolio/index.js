import React, { Component, Fragment } from 'react'

export default class Portfolio extends Component {
    render() {
        return (
            <div class="projects" id="projects">
                <div class="container-fluid">
                    <div class="row">
                        <div class="col-sm-12 col-md-12 text-center">
                            <div class="projects-header">
                                <h1>Projects and Portfolio</h1>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-sm-12 col-md-12 text-center">
                            <div class="projects-header-quote">
                                <h3>Sharing my endeavors and passions...</h3>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-sm-4 col-md-4 text-center">
                            <div class="luvtalk-project">
                                <h3>LUV TALK Website</h3>
                                <p>Website, created during my first internship, which utilized HTML, CSS, PHP, Ajax, Javascript/jQuery, and Wordpress. </p>
                                <div class="row">
                                    <div class="col-sm-4 col-md-4 col-sm-offset-4 col-md-offset-4">
                                        <div class="project-more-info">
                                            <a class="btn btn-default btn-border" href="#">More</a>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>

                        <div class="col-sm-4 col-md-4 text-center">
                            <div class="personal-website-project">
                                <h3>Personal Website</h3>
                                <p>Enjoyable side project that was created to experiment with more HTML and CSS, but also provided an outlet to showcase my abilities and interests.</p>
                                <div class="row">
                                    <div class="col-sm-4 col-md-4 col-sm-offset-4 col-md-offset-4">
                                        <div class="project-more-info">
                                            <a class="btn btn-default btn-border" href="#">More</a>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>

                        <div class="col-sm-4 col-md-4 text-center">
                            <div class="strike-zone-project">
                                <h3>Strike Zone Analysis</h3>
                                <p>Data analytics project completed during my time at the Illinois Math and Science Academy which studied the baseball strike zone based on the state of the game.</p>
                                <div class="row">
                                    <div class="col-sm-4 col-md-4 col-sm-offset-4 col-md-offset-4">
                                        <div class="project-more-info">
                                            <a class="btn btn-default btn-border" href="#">More</a>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-sm-2 col-md-2 col-sm-offset-5 col-md-offset-5">
                            <div class="see-more-button">
                                <a class="btn btn-default btn-border" href="#">More Projects</a>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        )
    }
}
