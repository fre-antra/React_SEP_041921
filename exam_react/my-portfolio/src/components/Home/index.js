import React, { Component, Fragment } from 'react';
import Contact from '../Contact';
import Info from '../Info/index';
import Portfolio from '../Portfolio';


export default class Home extends Component {
    render() {
        return (
            <Fragment>
                <Info />
                <Portfolio />
                <Contact />
            </Fragment>
        )
    }
}
