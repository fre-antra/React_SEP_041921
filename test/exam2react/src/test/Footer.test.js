import { render, fireEvent, waitFor, screen } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Footer from "../components/Footer/Footer";
import Home from "../components/Pages/Home/Home";
import Portfolio from "../components/Pages/Portfolio/Portfolio";
import Contact from "../components/Pages/Contact/Contact";

describe("Footer Component", () => {
  it("render without error", () => {
    const { container } = render(<Footer></Footer>);
    expect(container).toMatchSnapshot();
  });

  it("render with four icons", () => {
    render(<Footer></Footer>);
    const icon1 = screen.getByTestId("1");
    expect(icon1).toBeInTheDocument();
    const icon2 = screen.getByTestId("2");
    expect(icon2).toBeInTheDocument();
    const icon3 = screen.getByTestId("3");
    expect(icon3).toBeInTheDocument();
    const icon4 = screen.getByTestId("4");
    expect(icon4).toBeInTheDocument();
  });

  it("render with phone and email information", () => {
    render(<Footer></Footer>);
    const phone = screen.getByText(/Phone: XXXXXX/i);
    expect(phone).toBeInTheDocument();

    const email = screen.getByText(/Email: XXXXX@gmail.com/i);
    expect(email).toBeInTheDocument();
  });

  it("has five blocks of link", () => {
    render(<Footer></Footer>);
    const grid = screen.getByTestId("grid");
    expect(grid.childElementCount).toEqual(5);
  });

  it("brings user to Portfolio section when clicked on Portfolio link", async () => {
    render(<Footer></Footer>);
    const portfolio = screen.getByText(/Portfolio/i);
    expect(portfolio).toBeInTheDocument();

    fireEvent.click(portfolio);
    await render(<Portfolio></Portfolio>);
    expect(screen.getByTestId("portfolio-header")).toHaveTextContent(
      "Projects and Portfolio"
    );
  });

//  it("brings user to Contact section when clicked on Contact link", async () => {
//    render(<Footer></Footer>);
//    const contact = screen.getByText(/Contact/i);
//    expect(contact).toBeInTheDocument();

//    //fireEvent.click(contact);
//    //render(<Contact></Contact>)
//    console.log(contact.href);
//    window.location.href = contact.href;
//    console.log(window.location.href);
//    await waitFor(() => {
//        //console.log('log',screen.getByTestId("contact-header"));
//        //screen.getByTestId("contact-header")
//        //console.log(contact);
//      console.log(window.location.href);
//    });
//    expect(screen.getByText("Reaching out to me")).toBeInTheDocument()
//  });

  it("brings user to Home section when clicked on Resume link", async () => {
    render(<Footer></Footer>);
    const resume = screen.getByText(/Resume/i);
    expect(resume).toBeInTheDocument();

    fireEvent.click(resume);
    await render(<Home></Home>);
    expect(screen.getByTestId("home-section")).toBeInTheDocument();
  });
});
