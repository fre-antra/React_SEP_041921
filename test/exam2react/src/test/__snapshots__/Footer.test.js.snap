// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component render without error 1`] = `
<div>
  <div
    class="footer-section"
    data-testid="footer"
  >
    <div
      class="center-div undefined"
    >
      <div
        class="mt-50"
      />
      <div
        class="ui centered grid"
      >
        <div
          class="center-div undefined"
        >
          <div
            class="ui grid"
          >
            <div
              class="row"
            >
              <div
                class="four wide computer sixteen wide mobile column"
              >
                <img
                  alt="logo"
                  class="icon"
                  data-testid="1"
                  src="facebook.svg"
                />
              </div>
              <div
                class="four wide computer sixteen wide mobile column"
              >
                <img
                  alt="logo"
                  class="icon"
                  data-testid="2"
                  src="linkedin.svg"
                />
              </div>
              <div
                class="four wide computer sixteen wide mobile column"
              >
                <img
                  alt="logo"
                  class="icon"
                  data-testid="3"
                  src="pinterest.svg"
                />
              </div>
              <div
                class="four wide computer sixteen wide mobile column"
              >
                <img
                  alt="logo"
                  class="icon"
                  data-testid="4"
                  src="email.svg"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          class="row"
        >
          <div
            class="ui centered grid"
          >
            <div
              class="eight wide computer eight wide mobile eight wide tablet column"
            >
              <p>
                Phone: XXXXXX
              </p>
            </div>
            <div
              class="eight wide computer eight wide mobile eight wide tablet column"
            >
              <p>
                Email: XXXXX@gmail.com
              </p>
            </div>
          </div>
        </div>
        <div
          class="centered row"
        >
          <div
            class="center-div undefined"
          >
            <div
              class="ui grid"
              data-testid="grid"
            >
              <div
                class="three wide computer sixteen wide mobile column"
              >
                <div
                  class="footer-section__list"
                >
                  <a
                    class="footer-section__list-header"
                  >
                    XXXXXX
                  </a>
                  <a
                    href="#"
                  >
                    Copyright Â© 2021
                  </a>
                  <a
                    href="#"
                  >
                    NY
                  </a>
                </div>
              </div>
              <div
                class="three wide computer sixteen wide mobile column"
              >
                <div
                  class="footer-section__list"
                >
                  <a
                    class="footer-section__list-header"
                    href="#home"
                  >
                    Home
                  </a>
                </div>
              </div>
              <div
                class="three wide computer sixteen wide mobile column"
              >
                <div
                  class="footer-section__list"
                >
                  <a
                    class="footer-section__list-header"
                    href="#portfolio"
                  >
                    Portfolio
                  </a>
                  <a
                    href="#"
                  >
                    Project 1
                  </a>
                  <a
                    href="#"
                  >
                    Project 2
                  </a>
                  <a
                    href="#"
                  >
                    Project 3
                  </a>
                  <a
                    href="#"
                  >
                    See All
                  </a>
                </div>
              </div>
              <div
                class="three wide computer sixteen wide mobile column"
              >
                <div
                  class="footer-section__list"
                >
                  <a
                    class="footer-section__list-header"
                    href="#portfolio"
                  >
                    Resume
                  </a>
                  <a
                    href="#home"
                  >
                    Download
                  </a>
                </div>
              </div>
              <div
                class="three wide computer sixteen wide mobile column"
              >
                <div
                  class="footer-section__list"
                >
                  <a
                    class="footer-section__list-header"
                    href="#contact"
                  >
                    Contact
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
